### DISCUSSION TITLE
word heatmap based on WPMf for the exercice

### DISCUSSION THREAD

**Discussion by DasHammett** [monkeytypegame/monkeytype#1956] [Ideas]
I think it would be very useful to have a word heatmap no only with the burst wpm (current implementation), but also a heatmap based on the wpm for the exercice. This would allow you to understand which words are typed slower than the average wpm.

You would inmediately understand that red words are typed at a lower speed than the wpm of the exercice, white at aound the same speed and blue faster.

With the current implementation this is not possible, as some red words are typed faster than the wpm and this analysis cannot be done

Thanks

**Comment by Miodec** [comment]
Whats the advantage of seeing 'which words are typed slower than the average wpm'. You can just look for which words are slower overall, isn't that the same thing basically?

**Reply by DasHammett** [reply]
Slower compared to what? The current implementation is based on the burst of the word, but still is not clear to me to what we are comparing.

To me burst is not important, it's nice to see your burst, but I don't train for burst, so for me is more important to see which words are typed at a slower speed than the average.

**Reply by frogfrogfrogfrogfrog** [reply]
Isn't it compared to other words? If it's literally based off of burst words aren't the words being compared to each other? Such as how fast you burst one word compared to another word? As Miodec said couldn't you simply find the words that you typed slower than others? The concept would be the same in that you simply typed those words slower than you typed every other word (aka average).

**Reply by DasHammett** [reply]
> Isn't it compared to other words? If it's literally based off of burst words aren't the words being compared to each other? Such as how fast you burst one word compared to another word? As Miodec said couldn't you simply find the words that you typed slower than others? The concept would be the same in that you simply typed those words slower than you typed every other word (aka average).

If the words are compared among themselves, then the white words should be around the average, but that is not the case. As an exmple, I have an exercice with 68wpm with white words at 87wpm. It's quite a jump! I also have slow words at 72wpm which is a lot closer, but still above my average.

I can live with that, no problem whatsoever, it's just that I believe it is more useful to see the deviation per word against your average than the difference in burst for each word against "something" that we don't know what it is exactely.
